src/Components/UI/resultCount/ResultCount.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const ResultCount = ({ count }) => {
+  return (
+    <section className="container">
+      <div className="search-result">
+        <span>
```

Comment:
Whenver we can reduce the amount of html, it's better to do it. I don't think we need all this markup... if you need block-text inside semantic tag - better use <p> instead of combination of div and span

Also, it doesn't look that we need section for it as well - it doesn't really match semantic rule (sections are usually required to have role because they are too general) and all styles it has can be applied to p

In the end, it looks like we don't really need ResultCount content as a separate element